src/product/product.resolver.ts

```
@@ -13,8 +35,43 @@ export class ProductResolver {
+    const limit = pagination?.limit ?? 10;
+    searchDto.page = page;
+    searchDto.limit = limit;
+
+    // 검색 결과 가져오기
+    const items = this.productService.search(searchDto);
+
+    // 전체 상품 수 계산 (페이지네이션 없이 필터만 적용)
+    const searchDtoWithoutPagination = { ...searchDto };
+    delete searchDtoWithoutPagination.page;
+    delete searchDtoWithoutPagination.limit;
+    const totalItems = this.productService.search(searchDtoWithoutPagination);
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)